### ISSUE TITLE
readme issue

### ISSUE THREAD

**Issue by atstockland** [hirber/hirber#2]
The readme instructs to install "gem install hirb" ... but, should be "gem install hirber".  Thanks for this branch, it resolved the 2.7.2 issues I was having with Hirb.

**Comment by atstockland** [comment]
Im not sure I'm correct with "gem install hirber".  It successfully installed, but table views are standard irc, not nice like the old HIRB.

**Comment by panaak** [comment]
Having the same issue. In rails the tables for AR objects do not work.

**Comment by adlugopolski** [comment]
@atstockland @panaak Thanks for the feedback. I haven't really touched the content README, aside from adding a preface section. Name of the gem is changed, so one should use name `hirber` in both Gemfiles or using `gem install ...`. 

I am guessing the problem is, that the main gem entry file is still named `hirb`, so even though you need to install gem called `hirber`, you still need to `require "hirb"`. This should be fixed.

In order get things going try to do the following:

```ruby
require "hirb"
Hirb.enable
extend Hirb::Console
```

I also usually add the following to my `.irbrc` because I like to have tables made using unicode characters. This should not be necessary in order to get the things going though. Without this you should still get regular ASCII tables.

```ruby
%w[ActiveRecord::Base Hash Array].each do |klass|
  Hirb::Formatter
    .dynamic_config[klass] = {
      class: Hirb::Helpers::AutoTable,
      ancestor: true,
      options: { unicode: true },
    }
end
```

**Comment by panaak** [comment]
It it doesn't seem to work while also using `pry` resp. https://github.com/rweng/pry-rails

Without pry it works no problem. Any idea why? Can't investigate myself at this point.

**Comment by adlugopolski** [comment]
Try running console using `pry-rails`, then do: 

```ruby
require "hirb"
Hirb.enable
extend Hirb::Console
table [[1, 2], [2, 3]]
```
Does this produce a table?

**Comment by panaak** [comment]
Yes, it does.

```
[10] pry(main)> table [[1, 2], [2, 3]]
┌───┬───┐
│ 0 │ 1 │
├───┼───┤
│ 1 ╎ 2 │
│ 2 ╎ 3 │
└───┴───┘
2 rows in set
=> true
[11] pry(main)>
```

**Comment by adlugopolski** [comment]
I've released 0.8.2 version of the gem. The name was changed to "hirber", so there is no need for explicte require anymore. I've also updated the instruction in the README. I am planning a bigger update to the Readme, but I still need to fix some minor bugs first.

Looking forward for your feedback and suggestion for improvements. :-)

**Comment by adlugopolski** [comment]
@panaak I don't really use `pry-rails` so I don't know the internals. Getting the tables by default in `pry` probably requires future development. If you wish make a separate issue. I might look into this in the future.
